src/plugins/presentation_panel/public/panel_component/panel_header/presentation_panel_hover_actions.tsx

```diff
@@ -131,6 +119,21 @@ export const PresentationPanelHoverActions = ({
   const rightHoverActionsRef = useRef<HTMLDivElement | null>(null);
 
   const [combineHoverActions, setCombineHoverActions] = useState<boolean>(false);
+
+  const { euiTheme } = useEuiTheme();
+
+  const EDIT_MODE_OUTLINE = `${euiTheme.border.width.thin} dashed ${euiTheme.colors.borderBaseFormsControl}`;
```

Comment:
Can you use `useMemo` to avoid recreating these string on each render.

Reply:
I think `useMemo` usage is best left to things which are expensive to compute. `useMemo` itself can [introduce overhead ](https://refine.dev/blog/react-usememo/#use-usememo-only-for-expensive-calculations).